### PR TITLE
GitOps 1.7.5 release notes documantion.

### PR DESCRIPTION
**Pull request by LunguTheEngineer**
Jira issue : [RHDEVDOCS-5599](https://issues.redhat.com/browse/RHDEVDOCS-5599)

Aligned team: DevTools 
OCP Version(s): 4.10 and to 4.12

Docs preview: [63730--docspreview](https://63730--docspreview.netlify.app/openshift-enterprise/latest/cicd/gitops/gitops-release-notes#gitops-release-notes-1-7-5_gitops-release-notes)

SME review: @reginapizza 

QE review: @varshab1210 

Peer review: @Srivaralakshmi

### CODE REVIEW

**Comment by masauso-lungu** [comment]
Hello @reginapizza , the draft is ready, PTAL and let me know if there is more to be considered.

**Review by Srivaralakshmi — APPROVED** [review]
@masauso-lungu Approved! Nice work!

File: modules/gitops-release-notes-1-7-5.adoc [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+{gitops-title} 1.7.5 is now available on {product-title} 4.10, 4.11, and 4.12.
+
+[id="fixed-issues-1-7-5_{context}"]
+== Fixed issues
+
+The following issues have been resolved in the current release:
+
+* Before this update, there was a mismatch in the RSA key for known hosts in the `argocd-ssh-known-hosts-cm` config map. This update fixes the issue by matching the RSA key with the upstream project. Now, you can use the default RSA keys on default deployments. link:https://issues.redhat.com/browse/GITOPS-3239[GITOPS-3239]
+
+* Before this update, an old Redis image version was used when deploying the {gitops-title} Operator, which resulted in vulnerabilities. This update fixes the vulnerabilities on Redis by upgrading it to the latest version of the `registry.redhat.io/rhel-8/redis-6` image. link:https://issues.redhat.com/browse/GITOPS-3040[GITOPS-3040]
```
Comment (reginapizza) [inline modules/gitops-release-notes-1-7-5.adoc:19]: This should be https://issues.redhat.com/browse/GITOPS-3240 but 3040

**Pull request closed without merging**